Extensions/YandexGamesSDK.json

```diff
@@ -1009,7 +1009,7 @@
         {
           "codeOnly": false,
           "defaultValue": "",
-          "description": "Data key name (e.g. \"Lives\")",
```

Comment:
Any reason for changing this? It seems `e.g.` is a proper form, and a bit more explicit than writing "eg" (I don't see it often, and it looks like "egg" with a missing g :)). https://www.aje.com/arc/editing-tip-using-eg-and-ie/